src/vcSettings.h

```diff
@@ -59,6 +59,21 @@ enum
   vcMetadataMaxLength = 256,
 };
 
+enum vcMapTileServerFlag
+{
+	vcMTSF_Unknown,
```

Comment:
`Unknown` isn't required; defaulting to slippy is acceptable

Reply:
vcMapTileServerFlag gives the vcMTSF_Unknown type by default before it is used to better determine whether it is assigned to another type.